templates/private-net-existing.yaml

```diff
@@ -0,0 +1,19 @@
+heat_template_version: 2015-04-30
+
+# Template which provides a private network which is routed to an external
+# network. This template just passes through the name of an already created
+# private network.
+
+parameters:
+  external_net:
+    type: string
+    default: private
```

Comment:
This one too.  I wonder if we even need defaults in these files.  It will always be set by the parent template, right?